.codeclimate.yml

```diff
@@ -33,3 +33,5 @@ checks:
 exclude_patterns:
   - "ci_scripts/target_validation_report.py"
   - "tests/test_mbed_targets.py"
+  - "ci_scripts/sync_target_database.py"
```

Comment:
I listed them here because I think the ci scripts shouldn't be under the same level of autobot scrutiny as our actual production code. I don't know how to turn off checks in the CC UI.

Reply:
I'm not sure what's best here. The fact we can't do per-line ignores that are respected by the cli bothers me slightly. Ignoring entire files bothers me more.

I can show you how to turn off checks in CC UI.